### PR TITLE
Enable Splunk forwarder for Master's integration sandboxes

### PR DESCRIPTION
**Pull request by kdmccormick**
@edx/masters-neem 
@edx/devops 
https://openedx.atlassian.net/browse/EDUCATOR-4451

Depends on https://github.com/edx/sandbox-internal/pull/49

### CODE REVIEW

**Review by iloveagent57 — APPROVED** [review]
👍

**Review by schenedx — APPROVED** [review]
👍

**Comment by kdmccormick** [comment]
Not merging this yet; I think I also need to do a PR against edx/sandbox-internal to add the Splunk config values. EDIT: Updated the descrpition with said PR.

File: util/jenkins/ansible-provision.sh [hunk 1]
```diff
@@ -425,6 +425,13 @@ veda_pipeline_worker=${video_pipeline:-false}
 veda_encode_worker=${video_encode_worker:-false}
 video_pipeline_integration=${video_pipeline:-false}
 
+# ansible overrides for master's integration environment setup
+if [[ $registrar == "true" ]]; then
```
Comment (jdmulloy) [inline util/jenkins/ansible-provision.sh:429]: What happens if someone creates a regular (i.e. non-integration) sandbox with registrar?

Comment (kdmccormick) [reply util/jenkins/ansible-provision.sh:429]: Currently we've been running under the assumption that every sandbox with registrar should behave like an integration environment.

If that's a flawed assumption, we could move these behaviors under an `$masters_integration_env` option.

Comment (jdmulloy) [reply util/jenkins/ansible-provision.sh:429]: @kdmccormick I think that might be a good idea. There should probably be some sort of master switch to say this is an integration environment box and not a regular sandbox. At some point someone will probably want registrar on a normal sandbox.

Comment (kdmccormick) [reply util/jenkins/ansible-provision.sh:429]: Makes sense. I'll add a ticket to our backlog to add that switch.

**Pull request merged**